### PR TITLE
Include formatting imports.

### PR DESCRIPTION
**Pull request by ndmckinley**
This PR revises the rules for formatting to include sorted imports, sorts existing imports, and switches from using an sbt plugin to using the google java formatter directly in a docker container.  This gives us more control.  `bin/fmt` continues to work correctly.

### CODE REVIEW

**Comment by ndmckinley** [comment]
Fixes #23.

**Comment by natsid** [comment]
Thanks!!

Not related to this change, but related to the formatter in general: One thing I'm still unclear on is how to run the formatter myself (or whether I should expect things to happen automatically). Any tips?

**Review by natsid — APPROVED** [review]

**Comment by ndmckinley** [comment]
You just need to run `bin/fmt` and it should do it for you.  :)

**Comment by ndmckinley** [comment]
Added that to the README, thanks for pointing out it wasn't in there!

**Comment by natsid** [comment]
Excellent - thank you!

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ The client should be as simple as is practical to implement the desired user exp
 
 Java code should conform to the Google Java [styleguide](https://google.github.io/styleguide/javaguide.html). The project makes use of a linter and autoformatter for Java to help with this.
 
+We have an autoformatter if there isn't one in your IDE - just run `bin/fmt` and your code should be automatically formatted.
```
Comment (natsid) [inline README.md:25]: Might be good to mention we only have this for Java code (at least for now...).

I'd like to add something for our HTML and TS as well if possible. I'll look into it.

**Review by natsid — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ndmckinley) [reply README.md]: Done!

**Review by natsid — APPROVED** [review]

**Pull request merged**